src/app/src/mainwindow.cpp

```diff
@@ -505,6 +505,7 @@ void MainWindow::StartAcquisition(bool saveToDisk) {
             m_led = !m_led;
             double voltage = (m_config->ledIntensity / 100.0) * m_config->maxVoltage;
             spdlog::info("Setting led intensity {}, voltage {}, max voltage {}", m_config->ledIntensity, voltage, m_config->maxVoltage);
+            std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
The delay should be configurable through the nautilus.toml file and part of the Config object

Reply:
I think this delay should be added to the ledON method so that it doesn't get missed if we have another code path that turns on the led